components/wifi/esp8266-driver/mbed_lib.json

```diff
@@ -58,6 +67,14 @@
         "NUCLEO_F411RE": {
             "tx": "D8",
             "rx": "D2"
+        },
+        "NUMAKER_PFM_M2351_CM": {
+            "tx"                    : "PD_1",
+            "rx"                    : "PD_0",
+            "rst"                   : "PD_7",
+            "rts"                   : "PD_3",
+            "cts"                   : "PD_2",
+            "rst-pin-polarity"      : "1"
```

Comment:
Now that you didn't change the `rst-assert-time-us` configuration I second @VeijoPesonen 's remark - is this really necessary? If so - why not get the config for your target in here?
In general, I don't mind getting the module more configurable, but if the reset assertion time is a constant, it should stay that way.

Reply:
@michalpasztamobica Add `rst-assert-time-us` override for M2351. Its value follows our BSP sample code.